src/shims/windows/handle.rs

```diff
@@ -230,19 +230,11 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         let stderr = this.eval_windows("c", "STD_ERROR_HANDLE").to_i32()?;
 
         let fd_num = if which == stdin {
```

Comment:
Please add a comment saying where those numbers come from. AFAIK those are the typical Unix numbers, right? Miri just happens to set them up always, even on Windows.

Reply:
Will do. And yeah, this is just taking advantage of the existing setup. The user won't even see these values directly, due to handle encoding.

Reply:
If we added support for `SetStdHandle`, then we'd need to add state to track the current values of these handles. But since we don't support that, this will just only ever return these values. Adding such support is completely possible, but not really important for `std` as far as I can tell.